SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/VisibilityTest/VisibilityTest.cpp

```diff
@@ -79,6 +81,11 @@ void AVisibilityTest::PrepareTest()
 
 				FinishStep();
 			}
+
+			if (AssertTrue(HasAuthority(), TEXT("We should have authority over the test actor.")))
+			{
+				bRunningWithReplicationGraph = GetNetDriver()->GetReplicationDriver() != nullptr;
```

Comment:
Also why replicate this?

Reply:
Hm, I *think* the clients don't have the replication driver? So I have to populate this from a server to pass the information of "running with rep graph" down to the clients. I think.

Reply:
Yeah you're right - can just check `ReplicationDriverClassName` though

Reply:
Oh, that could be easier. Is that robust? I did this based on what we do in the NetDriver.

Reply:
Yep should be - it's just reading a ini config,

Reply:
Actually, what would happen if the replication graph class in the ini is not found?

Reply:
But I changed it, it now uses the class name.